media-gfx/gimp/gimp-2.99.6.ebuild

```diff
@@ -0,0 +1,255 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( luajit )
+PYTHON_COMPAT=( python3_{7..10} )
+GNOME2_EAUTORECONF=yes
+VALA_MIN_API_VERSION="0.40"
```

Comment:
please, bump vala min version to portage tree available now:  
`VALA_MIN_API_VERSION="0.44"`

Reply:
Wasn't sure about that one but it makes sense, done.